game/configitem.cc

```diff
@@ -193,19 +213,20 @@ std::string const ConfigItem::getValue() const {
 	throw std::logic_error("ConfigItem::getValue doesn't know type '" + m_type + "'");
 }
 
-void ConfigItem::addEnum(std::string name) {
+void ConfigItem::addEnum(std::string const& name) {
 	verifyType("uint");
-	if (find(m_enums.begin(),m_enums.end(),name) == m_enums.end()) {
+	if (find(m_enums.begin(), m_enums.end(), name) == m_enums.end())
 		m_enums.push_back(name);
-	}
+
 	m_min = static_cast<unsigned short>(0);
 	m_max = static_cast<unsigned short>(m_enums.size() - 1);
-	m_min = static_cast<unsigned short>(1);
```

Comment:
Facepalm. Good job catching that.

Reply:
Don't know how this could happen. My few unit test didn't catch it.